<launch>

<node pkg = "rospkg-ogameasure"
      name = "tr_73u"
      type = "tr_73u.py">

      <param name="~ondotori_usbport" value="/dev/ttyUSB0" />


  <remap from = '/dev/TandD/__port__/temperature' to = '/dev/TandD/ttyUSB0/temperature'/>
  <remap from = '/dev/TandD/__port__/humidity' to = '/dev/TandD/ttyUSB0/humidity'/>
  <remap from = '/dev/TandD/__port__/pressure' to = '/dev/TandD/ttyUSB0/pressure'/>


</node>

</launch>
